template/data/fr/pages.json

```
@@ -0,0 +1,308 @@
+      "contentSections": [
+        {
+          "__component": "sections.hero",
+          "title": "Le meilleure façon de faire votre site Corporate",
+          "label": "Nouveau Strapi starter",
+          "description": "Commencer vite avec votre site corporate",
+          "smallTextWithLink": "Vous voulez construire votre propre starter? Guide à venir",
+          "buttons": [
+            {
+              "url": "https://github.com/strapi/strapi-template-corporate",
+              "newTab": false,
+              "text": "Commencer",
+              "type": "primary"
+            },
+            {
+              "url": "https://github.com/strapi/strapi-template-corporate",
+              "newTab": true,
+              "text": "Voir le code",
+              "type": "secondary"
+            }
+          ],
+          "picture": null
+        },
+        {
+          "__component": "sections.feature-rows-group",
+          "features": [
+            {
+              "title": "Les équipes marketing peuvent maintenant faire leurs propre pages",
+              "description": "Grace à une liste pré-construite de blocs UI, les marketers peuvent faire la mise en page exactement comme ils/elles veulent",
+              "link": {
+                "url": "#",
+                "newTab": false,
+                "text": "Voir le liste de blocs UI"
+              },
+              "media": null
+            },
+            {
+              "title": "Laisser vos développeurs tranquille",
+              "description": "Vous pouvez publier, modifier, et supprimer des pages sans l'aide d'un développeur.  Votre équipe technique peut finalement concentrer sur d'autre taches.",
+              "link": {
+                "url": "#",
+                "newTab": false,
+                "text": "Voir comment créer une page"
+              },
+              "media": null
+            }
+          ]
+        },
+        {
+          "__component": "sections.feature-columns-group",
+          "features": [
+            {
+              "title": "Visualisez vos changments",
+              "description": "Grace à un mode preview, vous pouvez voir vos pages avant de les publier.",
+              "icon": null
+            },
+            {
+              "title": "Fully responsive",
+              "description": "Ce starter marche sur touts les tailles d'écrans.",
+              "icon": null
+            },
+            {
+              "title": "Facile à personnaliser",
+              "description": "On utilise Tailwind pour le styling. Vous pouvez changez la thème de votre site sans besoin d'aller dans le code.",
+              "icon": null
+            }
+          ]
+        },
+        {
+          "__component": "sections.testimonials-group",
+          "title": "Ici, les témoignages des vos clients.",
+          "description": "Cette partie est où vous pouvez montrer vos clients.  Mettez des citations, et montrez les logos des entreprises qui aiment votre produit.",
```

Rephrase because there's a word missing, it should be: `Cette partie est là/l'endroit où vous pouvez`

But it's lighter like this:

```suggestion
          "description": "Dans cette partie, vous pouvez mettre en avant vos clients.  Mettez des citations, et montrez les logos des entreprises qui aiment votre produit.",
```

(also avoids using montrer twice)